### PR TITLE
Subscribing to an API from application subscription page

### PR DESCRIPTION
**Pull request by ruwiniwj**
This PR enable subscribing to/unsubscribing from an API from the the subscriptions tab of an application.

### CODE REVIEW

**Comment by ruwiniwj** [comment]
Fix issue https://github.com/wso2/product-apim/issues/4969

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Listing/SubscriptionPolicySelect.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```
Comment (malinthaprasan) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Listing/SubscriptionPolicySelect.jsx]: License header

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Listing/SubscriptionPolicySelect.jsx]: Fixed

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/APIInfoDTO.java [hunk 2]
```diff
@@ -160,6 +163,24 @@ public void setThumbnailUri(String thumbnailUri) {
...
   }
 
+  /**
+   * List of throttling policies of the API
+   **/
+  public APIInfoDTO throttlingPolicies(List<String> throttlingPolicies) {
+    this.throttlingPolicies = throttlingPolicies;
+    return this;
+  }
+
```
Comment (vithu30) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/APIInfoDTO.java]: Shall we remove the empty lines/

Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/APIInfoDTO.java]: The DTO files are autogenerated.

**Pull request merged**